### DISCUSSION TITLE
brew install dieharder fails because cert is expired in chain

### DISCUSSION THREAD

**Discussion by zipzapzanigan** [Homebrew/discussions#3416] [Everyday usage]
```
❯ brew install dieharder -d
/opt/homebrew/Library/Homebrew/brew.rb (Formulary::FormulaLoader): loading /opt/homebrew/Library/Taps/homebrew/homebrew-core/Formula/dieharder.rb
Error: dieharder has been disabled because it uses an upstream server with an incomplete SSL certificate chain!
```
I believe this may be related to curl, but I can't for the life of me figure out how to get this to install properly. 
I tried prepending brew install with HOMEBREW_FORCE_BREWED_CURL=1  to no avail.
I also tried to set CURL_CA_BUNDLE to a different cert, to no avail.

the file homebrew-core/Formula/dieharder.rb  contains the following hints: 

```
  # At the time of writing (2022-01-17), the webhome.phy.duke.edu server has
  # an incomplete SSL certificate chain, which causes an error on Linux
  # and with brewed curl on macOS (`curl: (60) SSL certificate problem: unable
  # to get local issuer certificate`). We may be able to revert this deprecation
  # if this issue is fixed on the upstream server in the future.
  # Original deprecation date: 2022-01-19
```

There is a patch file that runs on linux, but it does not run on mac os.

I am running on Mac OS Monterey 12.4 on an Apple M1 Max

**Comment by carlocab** [comment]
The `dieharder` formula in homebrew/core has been disabled. This means you cannot use it to install `dieharder`.

Your alternatives are to install `dieharder` without using `brew`, or to create a version of the `dieharder` formula in your own tap to use to install.

You might run into the difficulties that resulted in disabling the formula when doing the above, though.
